README.md

```diff
@@ -37,14 +37,19 @@ var cjsTree = typescript(inputTree, {
 });
 ```
 
-The broccoli plugin class
+### Config Options:
+
+* `tsconfig:` See: https://www.typescriptlang.org/docs/handbook/tsconfig-json.html
+
+### Ways to use:
+
+via the broccoli plugin subclass
 
 ```js
 var TypeScript = require('broccoli-typescript-compiler').TypeScript;
 ```
 
-Legacy filter behavior (compile all .ts files and only .ts and
```

Comment:
the filter also doesn't support some of the features you can configure in tsconfig because it filters out all non `.ts` files.